.github/workflows/main.yml

```diff
@@ -9,6 +9,9 @@ on:
 env:
```

Comment:
The goal isn't just en but en-US, there are lots of formats of en locales..

The other PR was merged without that discussion, can you make that update in this PR for the karma config:
https://github.com/ni/nimble/pull/1381/files#diff-bc258864f2b77d1453fe6b926921f8db150e8e3ed98a5447dfb7a178dd0d2b7dR32